app/common/services/util.js

```diff
@@ -1,12 +1,17 @@
 module.exports = [
     '_',
     'CONST',
+    '$window',
 function (
     _,
-    CONST
+    CONST,
+    $window
 ) {
 
     var Util = {
+        currentUrl: function () {
```

Comment:
It looks like the `deploymentUrl` is going to end up wrong where its used in site settings too.. could you fix that up too?